app/controllers/comments_controller.rb

```diff
@@ -40,6 +40,7 @@ def create
     return if performed?
 
     @comment = @idea.comments.build(comment_params)
+    @comment.status_at_comment_time = Idea.statuses[@idea.status.to_sym]
     @comment.user = current_user
     render :new unless @comment.save
     return if performed?
```

Comment:
Rubocop is failing because this method is too big
> Metrics/AbcSize: Assignment Branch Condition size for create is too high. [18.68/15]
Is there a better way to do this?

Reply:
You can just move lines 42-44 to a `build_comment` method or something like that. :)